apps/site/navigation.json

```diff
@@ -20,6 +20,10 @@
       "link": "https://nodejs.org/docs/latest/api/",
       "label": "components.containers.navBar.links.docs"
     },
+    "support": {
+      "link": "https://nodejs.org/en/about/previous-releases",
```

Comment:
This needs to be a relative link. It's also unclear to me why we need to add previous releases to the top nav bar?

Reply:
thanks for calling this out. I did want to use a relative link, but must have just looked at the closest neighbor. will address.

As to having the link in the top nav bar, this is part of the requirements for participation in the ESP program. The page showing the support and EOL policy for the project needs to be easy to find for full participation, generally meaning part of the main navigation.